provisioning/openstack/README.md

```diff
@@ -1,3 +1,36 @@
 # ose3 OpenStack Provisioning
 
-Still in development, coming soon....
+## Client Tools Setup ##
+* Get Access to OS1: https://mojo.redhat.com/docs/DOC-28082#jive_content_id_Getting_Started
+* Install nova client package
+```
+yum install python-novaclient
+```
+* Log into the OpenStack Dashboard
+* Navigate to Compute > Access & Security > API Access and click to "Download OpenStack RC File"
+* Place rc file in openstack directory
+
+```bash
+mkdir ~/.openstack
+mv ~/Downloads/Consulting\ Middleware\ Delivery-openrc.sh ~/.openstack/openrc.sh
```

Comment:
This is specific for the "Consulting Middleware Delivery" rc file - maybe make the "mv" command a bit more generic?